ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendTestsIngestion.scala

```diff
@@ -93,20 +94,24 @@ private[backend] trait StorageBackendTestsIngestion
     partiesAfterLedgerEndUpdate should not be empty
   }
 
-  private val NumberOfUpsertPackagesTests = 100
+  private val NumberOfUpsertPackagesTests = 30
```

Comment:
lowered the total test-runs, because with the bigger package payloads now it takes some time